app/workers/evss/document_upload.rb

```diff
@@ -1,22 +1,33 @@
 # frozen_string_literal: true
 
+require 'ddtrace'
+
 class EVSS::DocumentUpload
   include Sidekiq::Worker
 
   # retry for one day
   sidekiq_options retry: 14, queue: 'low'
 
   def perform(auth_headers, user_uuid, document_hash)
-    Raven.tags_context(source: 'claims-status')
-    document = EVSSClaimDocument.new document_hash
+    document, client, uploader, file_body = nil
```

Comment:
Is there a reason you're first setting `document, client, uploader, file_body` to `nil`?

Reply:
Hey Holden! Yes, without explicitly defining the variables to begin with rubocop throws an error for having undefined variables since they are within the scope of various tracing blocks.